cmd/nerdctl/container/exec/exec.go

```diff
@@ -28,6 +28,10 @@ import (
 	"github.com/containerd/containerd/cio"
 	"github.com/containerd/containerd/cmd/ctr/commands"
 	"github.com/containerd/containerd/cmd/ctr/commands/tasks"
+	nerdClient "github.com/containerd/nerdctl/cmd/nerdctl/client"
+	"github.com/containerd/nerdctl/cmd/nerdctl/completion"
+	containerUtils "github.com/containerd/nerdctl/cmd/nerdctl/utils/container"
```

Comment:
This name is quite similar to https://github.com/containerd/nerdctl/tree/main/pkg/containerutil :)